projects/plugins/crm/package.json

```
@@ -1,29 +1,34 @@
 {
 	"name": "@automattic/jetpack-crm",
-	"version": "5.5.2-alpha",
+	"version": "5.6.0-alpha",
 	"description": "The CRM for WordPress",
 	"author": "Automattic",
 	"license": "GPL-2.0",
 	"scripts": {
-		"start": "npx wp-env start",
-		"stop": "npx wp-env stop",
-		"clean": "npx wp-env clean all"
+		"build": "pnpm run clean && pnpm run build-client",
+		"build-client": "NODE_PATH=\"$PWD/node_modules\" webpack",
+		"build-production": "NODE_ENV=production BABEL_ENV=production pnpm run build",
+		"clean": "rm -rf **/css/**/*.rtl.* .cache/ **/css/*.min.css **/js/*.min.js.* **/js/*.min.js  **/js/**/*.min.js.map **/css/**/*.css.map",
+		"watch": "pnpm run build && NODE_PATH=\"$PWD/node_modules\" webpack watch"
 	},
 	"repository": {
 		"type": "git",
 		"url": "https://github.com/Automattic/jetpack.git",
 		"directory": "projects/plugins/crm"
 	},
 	"devDependencies": {
-		"esm": "^3.2.25",
-		"gulp": "^4.0.2",
-		"gulp-autoprefixer": "^8.0.0",
-		"gulp-clean-css": "^4.3.0",
-		"gulp-rename": "^2.0.0",
-		"gulp-sass": "^5.0.0",
-		"gulp-uglify": "^3.0.2",
-		"sass": "^1.43.4",
-		"uglify-js": "^3.14.3"
+		"@automattic/jetpack-webpack-config": "workspace:*",
+		"@automattic/remove-asset-webpack-plugin": "workspace:*",
+		"@babel/core": "7.20.7",
+		"@babel/preset-env": "7.20.2",
+		"@babel/register": "7.18.9",
+		"@babel/runtime": "7.20.7",
```

Let's pick up the changes from #28682 here
```suggestion
		"@babel/core": "7.20.12",
		"@babel/preset-env": "7.20.2",
		"@babel/register": "7.18.9",
		"@babel/runtime": "7.20.13",
```